autopush/main.py

```diff
@@ -382,8 +382,9 @@ def endpoint_main(sysargs=None):
     args, parser = _parse_endpoint(sysargs)
     settings = make_settings(
         args,
-        endpoint_scheme="https" if args.ssl_key else "http",
-        endpoint_hostname=args.hostname,
+        endpoint_scheme=args.endpoint_scheme or "https"
+        if args.ssl_key else "http",
```

Comment:
flake8 was complaining when I had indentation, but I agree that it's confusing. Let me try a few things to see if I can make it better.